charts/platform-service/templates/virtualservice.yaml

```diff
@@ -27,24 +34,33 @@ spec:
         host: {{ include "platform-service.fullQualifiedServiceName" . | quote }}
 {{- include "retries" . }}
 
-{{- if .Values.gateway.exposeService }}
-
 ---
+{{- if .Values.gateway.exposeService }}
 apiVersion: networking.istio.io/v1alpha3
 kind: VirtualService
 metadata:
   name: {{ include "platform-service.serviceName" . }}-external
+  labels:
+{{ include "appLabels" . | trim | indent 4 }}
 spec:
   hosts:
-{{- if .Values.defaultRouting.allHosts }}
-  - "*"
-{{- else }}
-  {{- range .Values.defaultRouting.hosts }}
-  - {{ . | quote }}
+  {{- if .Values.defaultRouting.kubeitDefaultHosts.enabled }}

```

Comment:
We suggest moving these host names generation to the tenant app of apps and pass it a ready to use list here. 
So something like like 
{{- range .Values.defaultRouting.generatedHosts }}
    - {{ . | quote }}
{{- end }}
and populate generatedHosts in your "tenant bootstrap chart"(?).

Reply:
Might be a problem with passing the appname to a list, I'll have a look into it